go/vumitools/app_worker.py

```diff
@@ -198,7 +199,7 @@ def get_metadata_helper(self, msg):
             self.vumi_api, msg, conversation_cache=self._conversation_cache)
 
     @inlineCallbacks
-    def find_outboundmessage_for_event(self, event):
+    def _find_outboundmessage_for_event(self, event):
```

Comment:
I gave this method a leading underscore because it wasn't used anywhere outside this class and appears to be intentionally an internal method but I had to grep the code base to discover that. We probably also don't want people calling both `find_outboundmessage_for_event` and `find_message_for_event`.